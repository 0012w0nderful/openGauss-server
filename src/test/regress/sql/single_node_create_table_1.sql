
drop table if exists t000_t1;
create table t000_t1(f1 int, f2 int);
alter table t000_t1 set (append_mode=on, rel_cn_id=12345);
alter table t000_t1 set (append_mode=off);
drop table if exists t000_t1;
