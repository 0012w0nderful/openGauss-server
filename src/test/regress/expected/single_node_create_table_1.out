drop table if exists t000_t1;
NOTICE:  table "t000_t1" does not exist, skipping
create table t000_t1(f1 int, f2 int);
alter table t000_t1 set (append_mode=on, rel_cn_id=12345);
ERROR:  Alter-Table set append mode should have 'rel_cn_oid' set together
alter table t000_t1 set (append_mode=off);
ERROR:  Redistribution is not supported in single node
drop table if exists t000_t1;
